packages/db/fauna/resources/Index/batchEntryByBatchAndContent.js

```diff
@@ -0,0 +1,35 @@
+import fauna from 'faunadb'
+
+const {
+  CreateIndex,
+  Collection,
+  If,
+  Index,
+  Exists,
+  Not
+} = fauna
+
+/**
+ * Usage:
+ *
+ * Match(
+ *   Index('batchEntry_by_batch_and_content'),
+ *   Ref(Collection('Batch'), Var('batchId')),
+ *   Ref(Collection('Content'), Var('contentId'))
+ * )
+ */
+const index = {
+  name: 'batchEntry_by_batch_and_content',
+  source: Collection('BatchEntry'),
+  terms: [
+    { field: ['data', 'batch'] },
+    { field: ['data', 'content'] }
+  ]
+}
+
+// indexes cannot be updated
```

Comment:
What do you mean by this comment? We will be adding stuff incrementally ( over the course of a minute ) - will that not require an index update?

Reply:
Content of indexes can be updated - I mean you can't change what an index is indexing. You have to delete it and re-create or create one with a different name. Functions on the other hand _can_ be updated.